### PR TITLE
add how pki works topic

### PR DESCRIPTION
**Pull request by sfsmithcha**
Adds "how PKI works" to the "how swarm mode works guide.

Signed-off-by: Charles Smith charles.smith@docker.com

### CODE REVIEW

**Comment by sfsmithcha** [comment]
@diogomonica @vdemeester @mstanleyjones PTAL

File: docs/swarm/how-swarm-mode-works/pki.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+# How PKI works in swarm mode
+
+The swarm mode public key infrastructure (PKI) system built into Docker Engine
+makes it simple to securely deploy a container orchestration system. The nodes
+in a swarm use mutual Transport Layer Security (TLS) to authenticate, authorize,
+and encrypt the communications between themselves and other nodes in the swarm.
+
+When you create a swarm by running `docker swarm init`, the Docker Engine
+designates istself as a manager node. By default, the manager node generates a
```
Comment (diogomonica) [inline docs/swarm/how-swarm-mode-works/pki.md]: *itself

**Comment by diogomonica** [comment]
We could add more stuff, but LGTM!

**Comment by tonistiigi** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 👼 with @diogomonica's comment taken care of :wink:

**Pull request merged**